spec/features/idv/cancel_idv_step_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+feature 'cancel at IdV step', :idv_job do
+  include IdvStepHelper
+
+  context 'verify step' do
+    def complete_previous_idv_steps
+      sign_in_and_2fa_user(user_with_2fa)
+      visit verify_path unless current_path == verify_path
+    end
+
+    it_behaves_like 'cancel at idv step'
+    it_behaves_like 'cancel at idv step', :oidc
+    it_behaves_like 'cancel at idv step', :saml
+  end
+
+  context 'profile step' do
+    alias complete_previous_idv_steps start_idv_at_profile_step
```

Comment:
Hmm, 🤔, I went with alias because I thought it was less tricky. I think we could set a symbol in a `let` block though and call send on `IdvStepHelper`.